src/__tests__/ParentDashboardContainer.test.js

```diff
@@ -5,35 +5,38 @@ import { Provider } from 'react-redux';
 import { MemoryRouter } from 'react-router-dom';
 
 import { ParentLoadingComponent } from '../components/common';
-import ParentDashboardContainer from '../components/pages/ParentDashboard/ParentDashboardContainer';
+import NewParentDashboardContainer from '../components/pages/NewParentDashboard/NewParentDashboardContainer';
 
 const mockStore = configureStore([]);
 const store = mockStore();
 
 afterEach(cleanup);
-
-jest.mock('@okta/okta-react', () => ({
-  useOktaAuth: () => {
+jest.mock('@auth0/auth0-react', () => ({
+  Auth0Provider: ({ children }) => children,
+  withAuthenticationRequired: (component, _) => component,
+  useAuth0: () => {
     return {
-      authState: {
-        isAuthenticated: true,
-      },
-      authService: {
-        getUser: () => Promise.reject(),
+      isAuthenticated: false,
```

Comment:
I'm just curious about the change in state in testing. Previously the test was done with isAuthenticated being true and now it is false. Why the change?